README.md

```diff
@@ -54,7 +54,7 @@ First get all the requirements installed on your system.
 ### Prerequisites
 
 - [Node.js LTS (16.x)](https://nodejs.org/)
-- [Python 3.8.x](https://python.org/) or
+- [Python 3.8.x or later](https://python.org/)
```

Comment:
@stevepiercy We probably need to be more specific.

The supported Python versions depend on which version of the Plone backend is used. Plone 5.2 supports Python 3.6-3.8. Plone 6 supports Python 3.8-3.10. No Plone release officially supports (pre-release) Python 3.11 yet.

Reply:
Agreed, but I am unsure of how all the versions align. Is this correct? Volto alignment with Python and Plone is less clear to me.

| Plone | Python | Volto |
|---|---|---|
| 5.2 | 2.7, 3.6-3.8 | ?-15.0 |
| 6.0 | 3.8-3.10 | 15.0 |

Reply:
Are there earlier versions of Volto that work with Plone 5.2? If so, should we mention them?

Volto 16 is still alpha, and Plone 6 is beta. Should we mention those details, too?